server/views/pages/prisons/supported-prisons.njk

```diff
@@ -18,9 +20,55 @@
 
     <div class="govuk-grid-row" id="scan-barcode-form">
       <div class="govuk-grid-column-two-thirds">
-        <h1 class="govuk-heading-xl">Manage supported prisons - coming soon</h1>
+        <h1 class="govuk-heading-xl">Manage supported prisons</h1>
 
-        <p>Please select which prisons legal senders can send Rule39 mail to.</p>
+        <p>This is still a work in progress and does not actually change the legal sender prison lists yet.</p>
```

Comment:
I'm dubious of this, but happy if you are

Reply:
I don't actually have any users yet - and when I do it will just be the support team who are rolling out the application. So you're right I probably don't need it but it's just in case.